icu4c/source/common/unicode/unifilt.h

```diff
@@ -77,7 +78,7 @@ class U_COMMON_API UnicodeFilter : public UnicodeFunctor, public UnicodeMatcher
      * @return clone, or nullptr if an error occurred
      * @stable ICU 2.4
      */
-    virtual UnicodeFilter* clone() const = 0;
+    virtual UnicodeFilter* clone() const override { UPRV_UNREACHABLE; return nullptr; }
```

Comment:
This change a compilation time assertion/error to a run time error and I am concern about this.

Reply:
virtual UnicodeFilter* clone() const override = 0;

Reply:
I did this because the base class is cast to UnicodeFilter* somewhere in the code. I will try the override = 0; I hope it will work.